src/osdp_cp.c

```diff
@@ -1386,6 +1386,12 @@ int osdp_cp_send_command(osdp_t *ctx, int pd_idx, const struct osdp_cmd *cmd)
 	case OSDP_CMD_STATUS:
 		cmd_id = CMD_LSTAT;
 		break;
+	case OSDP_CMD_OUT_STATUS:
+		cmd_id = CMD_OSTAT;
+		break;
+	case OSDP_CMD_INPUT_STATUS:
+		cmd_id = CMD_ISTAT;
+		break;
```

Comment:
The output/input status have to be handled along with `case OSDP_CMD_STATUS:` above to avoid having to introduced too many command types. Also I think some refactoring around this area is overdue.

Please drop this change, I will take address it shortly.